### PR TITLE
[Overflow-237] feature/create-tooltip

### PR DESCRIPTION
**Pull request by kenttelebanco**
## Backlog Link
https://framgiaph.backlog.com/view/SUN_OVERFLOW-237
## Expected Output
Should display a tooltip when tag is selected
Should close once tag is selected again or clicked outside
## Notes
Installed React Popper package (Given an element, such as a button, and a tooltip element describing it, Popper will automatically put the tooltip in the right place near the button)
```
npm i react-popper @popperjs/core
```
## Screenshots
17 02 2023_17 03 54_REC

### CODE REVIEW

File: frontend/components/molecules/Tooltip/index.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import Button from '@/components/atoms/Button'
+import React from 'react'
+
+function Tooltips() {
```
Comment (nicoleamber) [inline frontend/components/molecules/Tooltip/index.tsx]: use arrow function expression

`const Tooltips = (): JSX.Element => {`

File: frontend/components/molecules/Tooltip/index.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            <a className="underline ml-2" href="">View Tag</a>
+            </p>
+        </div>
+        <div className="p-2 ml-2 float-right">
+            <Button
+            usage="popover"
+            type="submit"
+            onClick={undefined}
+            > Watch Tag
+            </Button>
```
Comment (nicoleamber) [inline frontend/components/molecules/Tooltip/index.tsx]: Apply proper formatting

**Review by nicoleamber — CHANGES_REQUESTED** [review]
Please run the prettier commands to format your changes.

`npm run format` - for checking
`npm run format:fix` - applying needed formatting

**Review by nicoleamber — APPROVED** [review]

**Review by presyos — APPROVED** [review]

**Comment by ipilikent** [comment]
## Approved PR
-  The tooltip is working
- The View Tag is displayed
- The Watch Tag button is displayed
 

## Proof
<details>

[237 - Markup Tooltip.webm](https://user-images.githubusercontent.com/110364019/220290595-e969f8c8-3787-498d-9501-47139e6e42a6.webm)

</details>

Test Case: https://docs.google.com/spreadsheets/d/114GrEn-LQQ8saIvjz9zqenuk05qmO4dN/edit#gid=48930172
-----
- Total Test time:  0.5 hr.
- Total Comments: 0
- Total Iterations: 0

**Pull request merged**